### PR TITLE
JAMES-3775 Experiment with Rspamd `Deliver-To` header

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
chibenwa/tmail-backend-distributed                       0.6.2-debug-6

File: third-party/rspamd/src/test/java/org/apache/james/rspamd/client/RspamdHttpClientTest.java [hunk 1]
```diff
@@ -201,6 +201,30 @@ void checkNonVirusMailUsingRspamdClientWithExactPasswordShouldReturnHasNoVirus()
         assertThat(analysisResult.hasVirus()).isFalse();
     }
 
+    @Test
+    void test() throws InterruptedException {
```
Comment (Arsnael) [inline third-party/rspamd/src/test/java/org/apache/james/rspamd/client/RspamdHttpClientTest.java:206]: Should not forget to change that name

**Review by Arsnael — COMMENTED** [review]
Read it, looks good to me so far

**Comment by chibenwa** [comment]
Rebased to solve the conflict.

I did not made progress on this topic just yet.

**Comment by vttranlina** [comment]
I rebased this pr & clean code in other pr https://github.com/apache/james-project/pull/1191
I don't see `Deliver-To` header work. 
I tried to call directly http request to Rspamd (by Postman), But I didn't see difference score when different `Deliver-To` header.
// I ensure the learning spam has been run

**Comment by vttranlina** [comment]
I created a issue on rspamd repository for help: https://github.com/rspamd/rspamd/issues/4269

**Pull request closed without merging**